src/functions.php

```diff
@@ -242,6 +239,13 @@ function get_exit_code(
     return $process->getExitCode() ?? 0;
 }
 
+function get_exit_code(...$args): int
+{
+    trigger_deprecation('jolicode/castor', '0.8', 'The "%s()" function is deprecated, use "Castor\%s()" instead.', __FUNCTION__, 'exit_code');
```

Comment:
I think we should require https://github.com/symfony/deprecation-contracts

Reply:
done